docs/compass/08-06-provisioner-clean-up-runtime-data.md

```diff
@@ -12,17 +12,23 @@ This tutorial shows how to clean up Runtime data. This operation removes a given
 To clean up Runtime data for a given Runtime, make a call to the Runtime Provisioner with a mutation like this:  
 
 ```graphql
-mutation { cleanupRuntimeData(id: "61d1841b-ccb5-44ed-a9ec-45f70cd1b0d3")}
+mutation { cleanupRuntimeData(id: "61d1841b-ccb5-44ed-a9ec-45f70cd1b0d3")
+	{
+    id
+    message
+  }
+}
 ```
 
-A successful call returns the ID of the wiped Runtime:
+A successful call returns the Runtime ID and the message on whether the data clean-up succeeded:
```

Comment:
```suggestion
A successful call returns the Runtime ID and the message on whether the data clean-up succeeded or not:
```

Reply:
I think the `or not` here is redundant and doesn't really add to the sound of it either, let's skip it :)